example/react/src/serviceWorker.js

```diff
@@ -21,7 +21,7 @@ const isLocalhost = Boolean(
 );
 
 export function register(config) {
-  if (process.env.NODE_ENV === 'production' && 'serviceWorker' in navigator) {
+  if (process?.env?.NODE_ENV === 'production' && 'serviceWorker' in navigator) {
```

Comment:
Hi @emschwartz this is a Create React App and process.env is defined by all CRA apps. So i don't think this change is required

Reply:
Oh right, I missed that that example was react-specific. I'll update the PR